downloadDirectory/src/main/java/org/reactome/release/downloadDirectory/Main.java

```diff
@@ -0,0 +1,223 @@
+				failedSteps.add("CreateReleaseTarball");
+				e.printStackTrace();
+			}
+		}
+		if (stepsToRun.contains("PathwaySummationMappingFile")) 
+		{
+			// This step takes all Human Pathway and creates a tab-separated file with columns containing the stableIdentifier, name, and summation of the instance
+			// Output: pathway2summation.txt
+			try {
+				PathwaySummationMappingFile.execute(dbAdaptor, releaseNumber);
+			} catch (Exception e) {
+				failedSteps.add("PathwaySummationMappingFile");
+				e.printStackTrace();
+			}
+		}
+		if (stepsToRun.contains("MapOldStableIds")) 
+		{
+			// This step iterates through all StableIdentifiers and maps them to the old Reactome ID in 'REACT_#####' format. Human instances are displayed first.
+			// Output: reactome_stable_ids.txt
+			try {
+				MapOldStableIds.execute(username, password, host, releaseNumber);
+			} catch (Exception e) {
+				failedSteps.add("MapOldStableIds");
+				e.printStackTrace();
+			}
+		}
+		// These file copy commands now use absolute paths instead of relative ones
+		if (stepsToRun.contains("gene_association.reactome")) 
+		{
+			// This step copies the gene_association.reactome file generated during the goa_prepare step of Release to the download_directory folder
+			// Output: gene_association.reactome
+			logger.info("Copying gene_association.reactome to release directory");
+			try {
+				Files.copy(Paths.get(releaseDirAbsolute + "goa_prepare/gene_association.reactome"), Paths.get(releaseNumber + "/gene_association.reactome"), StandardCopyOption.REPLACE_EXISTING);
+			} catch (Exception e) {
+				failedSteps.add("gene_association.reactome");
+				e.printStackTrace();
+			}
+		}
+		if (stepsToRun.contains("models2pathways.tsv"))
+		{
+			// This step copies the models2pathways.tsv file generated during the biomodels step of Release to the download_directory folder
+			logger.info("Copying models2pathways.tsv to release directory");
+			try {
+				Files.copy(Paths.get(releaseDirAbsolute + "biomodels/models2pathways.tsv"), Paths.get(releaseNumber + "/models2pathways.tsv"), StandardCopyOption.REPLACE_EXISTING);
+			} catch (Exception e) {
+				failedSteps.add("models2pathways.tsv");
+				e.printStackTrace();
+			}
+		}
+		if (stepsToRun.contains("CreateReactome2BioSystems")) 
+		{
+			// This step converts Reactome Pathway instances into the NCBI BioSystems format (http://www.ncbi.nlm.nih.gov/biosystems/). 
+			// Output: ReactomeToBioSystems.zip
+			// This zip file contains an 'xml' file for each species in Reactome in BioSystems format 
+			try {
+				CreateReactome2BioSystems.execute(host, database, username, password, port, releaseNumber);
+			} catch (Exception e) {
+				failedSteps.add("CreateReactome2BioSystems");
+				e.printStackTrace();
+			}
+		}
+
+		// Move files to downloadDirectory release folder
+		logger.info("Moving all generated files to " + releaseDownloadDirWithNumber);
+		File folder = new File(releaseNumber);
+		File[] releaseFiles = folder.listFiles();
```

Comment:
`folder.listFiles()` could return null and needs to be checked.  You could surround the for loop with `if (releaseFiles != null)`